lib/exe/indexer.php

```diff
@@ -199,6 +199,7 @@ function sendGIF(){
     header('Content-Length: '.strlen($img));
     header('Connection: Close');
     print $img;
+    ob_flush();
```

Comment:
we should use our tpl_flush() function instead (it flushes the output buffer (if any) and the buffer to the browser/webserver).